internal/services/cdn/cdn_frontdoor_firewall_policy_resource.go

```diff
@@ -533,7 +530,7 @@ func resourceCdnFrontDoorFirewallPolicyUpdate(d *pluginsdk.ResourceData, meta in
 	ctx, cancel := timeouts.ForUpdate(meta.(*clients.Client).StopContext, d)
 	defer cancel()
 
-	id, err := parse.FrontDoorFirewallPolicyID(d.Id())
+	id, err := parse.FrontDoorFirewallPolicyIDInsensitively(d.Id())
```

Comment:
this is coming from the config and so should be case-sensitive

```suggestion
	id, err := parse.FrontDoorFirewallPolicyID(d.Id())
```